rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandlerSpec.groovy

```diff
@@ -354,6 +358,71 @@ class AWSBakeHandlerSpec extends Specification {
       1 * packerCommandFactoryMock.buildPackerCommand("", parameterMap, "$configDir/$awsBakeryDefaults.templateFile")
   }
 
+  void 'produces packer command with all required parameters for amzn, with sudo using runAsRoot:true'() {
+    setup:
```

Comment:
The CloudProviderBakeHandler is an abstract class. I think it must be extended to be tested, so I choose to use AWSBakeHandler to test it. I could probably have done this differently, but I don't know how a better solution would be. Imho the test adds value because it is testing the getBaseCommand in the abstract class.